fashion-example-query/README.md

```diff
@@ -3,83 +3,126 @@
 **Table of Contents**
 
 - [Run the Hello World example using query language](#run-the-hello-world-example-using-query-language)
-  - [🗝️ Key Concepts](#️-key-concepts)
-  - [🏃 Run the Flows](#-run-the-flows)
-  - [Results](#results)
-  - [QueryLanguage](#querylanguage)
+  - [🗝️ Expected knowledge](#Expected-knowledges)
+  - [🔮 Overview of the files](#Overview-of-the-files)
+  - [🏃 Run the Flows](#run-the-flows)
+  - [🌟 Results](#results)
+  - [🧞‍QueryLanguage](#querylanguage)
+  - [💫 Deploy with Docker](#Deploy-with-docker)
+  - [Next steps](#next-steps)
+  - [Community](#community)
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
-# Run the Hello World example using query language
+# Run Fashion-MNIST Example Using Query Language
 
-## 🗝️ Key Concepts
+This is an extension of the Hello World example using QueryLanguage. 
+We will use the FashionMNIST dataset to index and query. The dataset has different categories such as t-shirts, trousers, and pullovers. We will use QueryLanguage to create a different index per each category so we can query through them. 
 
-First of all, read up on [Jina 101](http://101.jina.ai) so you have a clear understanding of how Jina works. We're going to refer to those concepts a lot. We assume you already have some knowledge of Python and machine learning.
 
-This is an extension of the Hello World example, but here we use QueryLanguage to create an index per category and then, with that we can filter by category
+## 🗝️ Expected knowledge 
 
+1. Read up on [Jina 101](http://101.jina.ai). 
+2. Python knowledge.
+3. You have already run the [Hello World Example](https://docs.jina.ai/chapters/helloworld/index.html) successfully.
 
 
-### Install Requirements
+### Install requirements
 
-In your terminal:
+There are some requirements to run this example, to download everything you need, just run in your terminal:
 
 ```
 pip install -r requirements.txt
 ```
 
+## 🔮 Overview of the files
+
+* flows/index.yml: YAML file for indexing.
+* flows/query.yml: YAML file for querying.
+* pods/encoder.yml: YAML file for encoder pod.
+* pods/indexer-pullover.yml: Indexer for each specific category.
+* /workspace: Directory that stores indexed files (embeddings and documents). Automatically created after the first indexing.
+* Dockerfile: Dockerfile to run the example.
 
 ## 🏃 Run the Flows
 
-Now that we've got the code to load our data, we're going to dive into writing our app and running our Flows!
+We usually need at least two Flows in Jina. One for Indexing and one for Querying. 
 
 ### Index Flow
 
-To run the index you type:
+First thing we need is to index our data, and we need a Flow to manage this. 
```

Comment:
I think it would help the user by clarifying which file we are working on at the moment. To be specific that all the following functions are contained in app.py which in turn reference specific YAML configuration files.